### PR TITLE
Refactor pipeline to workflow in database analysis table

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

- Refactor pipeline, and pipeline_version to workflow in database analysis table

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b refactor-pipeline-to-workflow -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
Db and API 👍 
Screenshot 2024-02-28 at 10 56 45

File: tests/store/crud/read/test_read.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def test_analyses_to_upload_when_filtering_with_workflow(helpers, sample_store,
 
     for analysis_obj in records:
         # THEN the workflow should be MIP in the analysis object
-        assert analysis_obj.pipeline == str(pipeline)
+        assert analysis_obj.workflow == str(pipeline)
```
Comment (ChrOertlin) [inline tests/store/crud/read/test_read.py:258]:
```suggestion
        assert analysis_obj.workflow == str(workflow)
```

**Review by ChrOertlin — APPROVED** [review]
Looks good.

**Review by islean — APPROVED** [review]
LGTM!

File: tests/store/crud/read/test_read.py (hunk 1 above)
Comment (henrikstranneheim) [reply tests/store/crud/read/test_read.py]: Fixed

**Comment by henrikstranneheim** [comment]
Deployed to production:

```
Log deploy... done.
cg, version 59.5.9

**Pull request merged**